### DISCUSSION TITLE
TBS Unify Pro 32 + INAV 5.0 - impossible to set vtx power level to max (also tried with SA)

### DISCUSSION THREAD

**Discussion by xander1990** [iNavFlight/inav#8380] [General]
Hello everybody! 

I have an issue with my brand new 7" that runs INAV 5.0. The FC is a Matek F722 se and the VTX is a TBS Unify Pro. 
I have followed a couple of tutorials to unlock it and set it to max power, but when I go on the OSD and enter the SA menu I see the top number as 25, and if I try to change it to 800mw with the sticks (hitting save + reboot), the top number still stays stuck to 25. If I go directly to INAV via USB and set it to power level of 4 or 5 still it stays to 25. 

What could it be the issue? If you need any screenshots or pictures of the quad let me know! Thanks to everybody who's willing to help, this problem is driving me nuts

**Comment by breadoven** [comment]
Is `vtx_low_power_disarm ` active ?

**Reply by xander1990** [reply]
It's set on Until first arm
